src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -77,13 +78,28 @@ type Options<P> = {
   setNativeProps: (ref: AnimatedComponentRef, props: P) => void;
 };
 
+function assertComponentIsComponent(
```

Comment:
What is this assert for?

Reply:
If we want to add this overload we have to include `typeof FlatList` in definition of `createAnimatedComponent`. Then it starts to throw error since in fact `ComponentType<InitialComponentProps>` has some props that FlatList don't.